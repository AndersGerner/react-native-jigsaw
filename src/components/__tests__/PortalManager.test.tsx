import * as React from "react";
import renderer from "react-test-renderer";
import PortalManager from "../Portal/PortalManager";

it("renders portal manager", () => {
  const tree = renderer.create(<PortalManager />).toJSON();

  expect(tree).toMatchSnapshot();
});
