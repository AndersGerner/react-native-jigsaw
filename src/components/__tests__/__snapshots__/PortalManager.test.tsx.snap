// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders portal manager 1`] = `null`;
